src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,582 @@
+        if cbsd_in_nbrhd and freq_check:
+            cbsd_grant = CBSDGrant(
+                # Get information from the registration request
+                latitude=lat_cbsd,
+                longitude=lon_cbsd,
+                height=height_cbsd,
+                indoorDeployment=reg_request[i].get('installationParam',{}).get('indoorDeployment'),
+                antennaAzimuth=reg_request[i].get('installationParam',{}).get('antennaAzimuth'),
+                antennaGain=reg_request[i].get('installationParam',{}).get('antennaGain'),
+                antennaBeamwidth=reg_request[i].get('installationParam',{}).get('antennaBeamwidth'),
+                # Get information from the grant request
+                grantIndex=grant_index[i],
+                maxEirp=grant_request[i].get('operationParam',{}).get('maxEirp'),
+                lowFrequency=low_freq_cbsd,
+                highFrequency=high_freq_cbsd)
+
+            grants_inside.append(cbsd_grant)
+
+    return grants_inside
+
+def ComputeOOBConductedPower(low_freq_cbsd, low_freq_c, high_freq_c):
+
+    """
+    Compute maximum conducted power of a CBSD grant to an out-of-band
+    protection constraint based on FCC Part 96 Rules (96.41)
+
+    Inputs:
+        low_freq_cbsd:  low frequency of CBSD grant channel (Hz)
+        low_freq_c:     low frequency of protection constraint (Hz)
+        high_freq_c:    high frequency of protection constraint (Hz)
+
+    Returns:
+        Maximum out-of-band conducted power (dBm)
+    """
+
+    # Compute overlapping bandwidths
+    overlap_bw_within_10MHz = min(low_freq_cbsd, high_freq_c) \
+                              - max(low_freq_cbsd-10000000.0, low_freq_c)
+    overlap_bw_outside_10MHz = min(low_freq_cbsd-10000000.0, high_freq_c) \
+                               - max(3530000000.0, low_freq_c)
+    overlap_bw_below_3530MHz = min(3530000000.0, high_freq_c) \
+                               - low_freq_c
+
+    # Compute total conducted power
+    power_within_10MHz_mW = 0
+    power_outside_10MHz_mW = 0
+    power_below_3530MHz_mW = 0
+    if overlap_bw_within_10MHz > 0:
+        power_within_10MHz = OOB_POWER_WITHIN_10MHZ \
+                             + 10 * np.log10(overlap_bw_within_10MHz / 1000000.0)
+        power_within_10MHz_mW = np.power(10.0, power_within_10MHz/10.0)
+    if overlap_bw_outside_10MHz > 0:
+        power_outside_10MHz = OOB_POWER_OUTSIDE_10MHZ \
+                              + 10 * np.log10(overlap_bw_outside_10MHz / 1000000.0)
+        power_outside_10MHz_mW = np.power(10.0, power_outside_10MHz/10.0)
+    if overlap_bw_below_3530MHz > 0:
+        power_below_3530MHz = OOB_POWER_BELOW_3530MHZ \
+                              + 10 * np.log10(overlap_bw_below_3530MHz / 1000000.0)
+        power_below_3530MHz_mW = np.power(10.0, power_below_3530MHz/10.0)
+
+    power_mW = power_within_10MHz_mW + power_outside_10MHz_mW + power_below_3530MHz_mW
+
+    return 10 * np.log10(power_mW)
```

Comment:
Missing the overall scalor from the cbsd power (ie something like (max_cbsd_eirp - max_ant_gain)).

Reply:
Ok I see that you use something similar to the FSS out of band purge list, where the power is the absolute power in dBm/MHz that the CBSD cannot exceed. (as defined in R0-DEV-4 and R0-DEV-5).
Where in a spec or doc, do we have this way of doing defined ?